forms-thanks/annex-request-thanks.html

```diff
@@ -1,36 +1,86 @@
-<!doctype html>

-<html lang="en">

-<head>

-  <title>Thanks: Library Storage Annex: MIT Libraries</title>

-  <link href="/css/header-semantic.css" rel="stylesheet" type="text/css">

-  <link href="/css/wp-libraries.css" rel="stylesheet" type="text/css">

-  <script type="text/javascript" src="/scripts/googleanalytics-async.js"></script>

-</head>

-

-<body>

-  <div id="container">

-    <!--#include virtual="/includes/header-semantic.html"-->

-      <!-- breadcrumb - TODO: make this semantic -->

-      <div class="breadcrumb">

-      <span class="semantic">You are located in:</span>

-      <ul>

-        <li><a href="/"><span class="semantic"></span>MIT Libraries</a></li>

-        <li><a href="/locations"><span class="semantic">in subsection </span>Locations</a></li>

-        <li><a href="/lsa"><span class="semantic">in subsection </span>Library Storage Annex</a></li>

-        <li><a href="/forms/annex-request.html"><span class="semantic">in subsection </span>Annex Request Form</a></li>

-        <li><span class="semantic">on page </span>Thank you</li>

-      </ul>

-      </div>

-    <h1>Annex Request</h1>

-    <p>Thank you! Your request is being forwarded to the <a href="../lsa/index.html">MIT Libraries' Library Storage Annex.</a> You may pick it up at the circulation desk in the library you selected. Meanwhile, you will receive confirmation by email. </p>

-    <p>If you don't receive  confirmation, it's possible that your request didn't go through. In that case, please fill out the form again, or <a href="http://libraries.mit.edu/about/contact.html">contact us</a>.</p>

-    <ul>

-      <li>Materials are normally delivered within 24 hours or less. </li>

-      <li>You should receive email notification when your items arrive at the destination library.</li>

-      <li>Anyone may call the <a href="../about/index.html">destination library</a> to verify that an item has arrived.&nbsp;</li>

-    </ul>

-    <p><a href="/forms/annex-request.html">Make another request</a></p>

-    <!--#include virtual="/includes/footer-semantic.html" -->

-  </div>

-</body>

-</html>

+<!DOCTYPE html>
+<!--[if lte IE 9]><html class="no-js lte-ie9" lang="en"><![endif]-->
+<!--[if !(IE 8) | !(IE 9) ]><!-->
+<html lang="en-US" class="no-js">
+<!--<![endif]-->
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Thanks: Library Storage Annex: MIT Libraries</title>
+
+  <!--#include virtual="/mitlib-style/inc/assets-site-head.html" -->
+
+  <!-- form specific stuff -->
```

Comment:
Removed the box styles for the thanks pages. The comment seems to be asking to removing the assets-site-head.html but that would cause the page to be not formatted correctly